tests/app/db.py

```diff
@@ -239,6 +239,7 @@ def create_notification(
     reply_to_text=None,
     created_by_id=None,
     postage=None,
+    save_notification=True,
```

Comment:
It looks here that the need is more to get a notification rather than to create one. I think there are other functions we could reuse to get a notification without going through this one, is it possible? Might be we can sync to see the existing ones?

Reply:
removed